fail-mbr/compile-mbr.sh

```diff
@@ -1,56 +0,0 @@
-
-srcdir="$(dirname "$0")"
-
-hostarch="$(dpkg-architecture -q DEB_BUILD_ARCH 2>/dev/null)" \
-|| hostarch="$(rpm --eval '%{_arch}' 2>/dev/null)" \
-|| hostarch="$(arch 2>/dev/null)" \
-|| hostarch=''
-
-case "${hostarch:-}" in
-'i386'|'i486'|'i586'|'i686'|'amd64'|'x86_64')
-
-    ############### compiles source file for x86 architectures ##################
-    # compile the file fail-mbr.bin
-    #echo -n "Compiling: fail-mbr.S -> fail-mbr.o -> "
-    #gcc -Wall -Werror -m32 -nostdlib -static -Wl,--relocatable -o fail-mbr.o fail-mbr.S
-
-    #echo -n "fail-mbr.image -> "
-    #gcc -Os -Wall -W -Wshadow -Wpointer-arith -Wmissing-prototypes -Wundef -Wstrict-prototypes -g -falign-jumps=1 -falign-loops=1 -falign-functions=1 -mno-mmx -mno-sse -mno-sse2 -mno-3dnow -fno-dwarf2-cfi-asm -fno-asynchronous-unwind-tables -m32 -fno-stack-protector -mno-stack-arg-probe -Werror -Wno-trampolines -DUSE_ASCII_FAILBACK=1 -DHAVE_UNIFONT_WIDTHSPEC=1  -mrtd -mregparm=3       -fno-builtin   -m32 -Wl,--build-id=none   -nostdlib -Wl,-N,-S -Wl,-N -Wl,-Ttext,0x7C00   -Wl,--no-dynamic-linker   -o fail-mbr.image fail-mbr.o
-
-    #echo "fail-mbr.bin [Done]. "
```

Comment:
Why did we loose this? These steps are important to know an have, if we are in a similar situation like at #211 again…

Reply:
Sorry, it wasn't clear to me why this was commented out. I figured it was just unused. Please could you explain. We could add it back, perhaps as Makefile rules that don't get called automatically.

Reply:
I guess it's for targeting 32-bit on 64-bit, but there's only one fail-mbr.bin.orig to compare against?

Reply:
Sorry, I'm not an expert at this. I only can tell that the commented steps helped me to track down the difference step by step when using older and newer tools.

Reply:
Putting aside the suggestion that this can be dropped, I do find it a little surprising that you can seemingly always get the same result as fail-mbr.bin.orig without all those extra flags. Since that does seem to work, you can just run `configure` followed by `make -C fail-mbr fail-mbr.bin`, which will simply build it for you. If you need to tweak the flags, just copy the output and adjust or edit Makefile.am as required.